src/lib/fixers.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable no-console, max-params */
+const { request } = require('@octokit/request');
+
+async function inviteUser(username, organization, role = 'member', token, dryRun) {
```

Comment:
imho you shouldn't disable the max params.
also that default of `role = 'member'` in the middle doesn't really work like it typically would, it would only get used if `undefined` gets passed to this function, but this function appears to require a `token` parameter, so you can't just call `inviteUser(name, org)`

Reply:
Oh yeah good call on this. I knew it felt/looked a little weird when I was writing it. I'll get that fixed up!

Reply:
Also I'm open to not disabling `max-params`, but I don't have another solution readily available for fixing up the `inviteUser` function. I think long-term this would call for a config object, right?

Reply:
yeah, you'd do this an options object/named parameters so this would become:

```js
async function inviteUser({ username, organization, role = 'member', token, dryRun }) {
```

then you'd just call it similarly:

```js
await inviteUser({
  username: 'whatever',
  organization: 'some-org'
});
```

Reply:
Interesting, what is the functional advantage of doing this over the way it currently is? I see that it looks more explicit which I'm generally in favor of. I'm just curious if there is any other advantage, aside from avoiding the `max-params`.

Reply:
The idea behind the `max-params` eslint rule is that is becomes hard to reason about an api with too many parameters when reading and writing code that calls the function. "is the third parameter the org name or the user name". with an options object there's no ambiguity there's a property called `organization` and that's what its for.  It does introduce a bit of ambiguity of which properties are actually required, but really that already exists with optional parameters and JavaScript in general.

Reply:
@msluther Done! All the fixers now use config objects. Thanks for working with me through that.